libs/ngx-dsfr/upload/src/lib/testing/upload.component.spec.ts

```diff
@@ -0,0 +1,103 @@
+/**
+ * Angular imports
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { By } from '@angular/platform-browser';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { FormsModule } from '@angular/forms';
+
+/**
+ * 3rd-party imports
+ */
+import { RouterLinkDirectiveStub } from '@betagouv/ngx-dsfr/testing';
+
+/**
+ * Internal imports
+ */
+import { TestHostComponent } from './test-host.component';
+import { DsfrUploadComponent, EMPTY_ID_ERROR, EMPTY_NAME_ERROR } from '../upload.component';
+import { DsfrUploadHarness } from './upload.harness';
+
+describe('DsfrFooterComponent', () => {
```

Comment:
`DsfrUploadComponent` ... please, be more rigorous next time... I should not be using my time in Code Reviews for that kind of "copy/paste" mistakes 😒